packages/visual-editor/src/docs/hybrid-development.md

```diff
@@ -149,14 +149,14 @@ Various existing components require additional data from the hybrid-developer. T
 This could look like:
 
 ```ts
-const resolveUrlTemplate = (
-  streamDocument: StreamDocument,
+function resolveUrlTemplate(
```

Comment:
Can we change this back to the other version? Both here and the actual function.